miasm2/jitter/loader/pe.py

```diff
@@ -162,8 +162,7 @@ def vm_load_pe(vm, fdata, align_s=True, load_hdr=True, name="", **kargs):
                 new_size = pe.SHList[i + 1].addr - section.addr
                 section.size = new_size
                 section.rawsize = new_size
-                section.data = strpatchwork.StrPatchwork(
-                    section.data[:new_size])
+                section.data = section.data[:new_size]
```

Comment:
This change seems not related to the PR. Is is wanted?

Reply:
done